phpunit.xml

```diff
@@ -8,7 +8,6 @@
          convertWarningsToExceptions="true"
          processIsolation="false"
          stopOnFailure="false"
-         syntaxCheck="false"
```

Comment:
That's because newer versions of PHPUnit 7.x removed that setting, right?

Reply:
Yes correct, there we're testing problems with the 7.0.x releases and later PHP7 versions (deprecation of `__toString()` on `ReflectionType` errors which could not be suppressed)